src/codemagic/tools/app_store_connect/action_groups/bundle_ids_action_group.py

```diff
@@ -0,0 +1,165 @@
+        deprecation_info=cli.ActionDeprecationInfo("create-bundle-id", "0.49.0"),
+    )
+    def create_bundle_id(
+        self,
+        bundle_id_identifier: str,
+        bundle_id_name: Optional[str] = None,
+        platform: BundleIdPlatform = BundleIdPlatform.IOS,
+        should_print: bool = True,
+    ) -> BundleId:
+        """
+        Create Bundle ID in Apple Developer portal for specifier identifier
+        """
+
+        if bundle_id_name is None:
+            bundle_id_name = bundle_id_identifier.replace(".", " ")
+
+        create_params = dict(identifier=bundle_id_identifier, name=bundle_id_name, platform=platform)
+        return self._create_resource(self.api_client.bundle_ids, should_print, **create_params)
+
+    @cli.action(
+        "get",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("get-bundle-id", "0.49.0"),
+    )
+    def get_bundle_id(
+        self,
+        bundle_id_resource_id: ResourceId,
+        should_print: bool = True,
+    ) -> BundleId:
+        """
+        Get specified Bundle ID from Apple Developer portal
+        """
+
+        return self._get_resource(bundle_id_resource_id, self.api_client.bundle_ids, should_print)
+
+    @cli.action(
+        "list",
+        BundleIdArgument.BUNDLE_ID_IDENTIFIER_OPTIONAL,
+        BundleIdArgument.BUNDLE_ID_NAME,
+        BundleIdArgument.PLATFORM_OPTIONAL,
+        BundleIdArgument.IDENTIFIER_STRICT_MATCH,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("list-bundle-ids", "0.49.0"),
+    )
+    def list_bundle_ids(
+        self,
+        bundle_id_identifier: Optional[str] = None,
+        bundle_id_name: Optional[str] = None,
+        platform: Optional[BundleIdPlatform] = None,
+        bundle_id_identifier_strict_match: bool = False,
+        should_print: bool = True,
+    ) -> List[BundleId]:
+        """
+        List Bundle IDs from Apple Developer portal matching given constraints
+        """
+
+        def predicate(bundle_id):
+            return bundle_id.attributes.identifier == bundle_id_identifier
+
+        bundle_id_filter = self.api_client.bundle_ids.Filter(
+            identifier=bundle_id_identifier,
+            name=bundle_id_name,
+            platform=platform,
+        )
+        bundle_ids = self._list_resources(
+            bundle_id_filter,
+            self.api_client.bundle_ids,
+            should_print,
+            filter_predicate=predicate if bundle_id_identifier_strict_match else None,
+        )
+
+        return bundle_ids
+
+    @cli.action(
+        "delete",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        CommonArgument.IGNORE_NOT_FOUND,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("delete-bundle-id", "0.49.0"),
+    )
+    def delete_bundle_id(
```

Comment:
Previously defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L458-L472